journal/original/20-chapter-17.adoc

```diff
@@ -889,6 +914,7 @@ cause of Christ.`"
      Of human nature--who appeared as meek
      And harmless as an infant--was compelled
      To dwell in loathsome prisons.`" `
+     // ^^ Does this need to be formatted differently?
```

Comment:
This is OK as is, it's a very rare thing -- poetry inside a footnote, and it looks weird, but it is formatted correctly.